src/main/kotlin/dto/Credential.kt

```diff
@@ -17,5 +17,13 @@ data class Credential(
     var senderName: String? = null,
     var projectName: String? = null,
     var applicationId: String? = null,
-    var clientId: String? = null
+    var clientId: String? = null,
+    var requireTls: Boolean = false,
+    var ignoreTls: Boolean = false,
+    var baseUrl: String? = null,
+    var webhookUrl: String? = null,
+    var redirectUrl: String? = null,
+    var hmac: Boolean = false,
```

Comment:
Kindly make all the Boolean fields nullable and they should have a default value of null

Reply:
So, API can omit these fields. OK, I'll update it